mantidimaging/gui/windows/live_viewer/view.py

```diff
@@ -34,6 +37,23 @@ def __init__(self, main_window: 'MainWindowView', live_dir_path: Path) -> None:
         self.live_viewer = LiveViewWidget()
         self.imageLayout.addWidget(self.live_viewer)
         self.live_viewer.z_slider.valueChanged.connect(self.presenter.select_image)
+        self.image_rotation_angle = 0
+        self.right_click_menu = self.live_viewer.image.vb.menu
+        rotate_menu = self.right_click_menu.addMenu(translate("ViewBox", "Rotate Image"))
+        rotate_angles_group = QActionGroup(self)
+        rotate_0 = QAction(translate("ViewBox", "0" + u'\N{DEGREE SIGN}'), rotate_angles_group)
```

Comment:
Given that we are not using translation anywhere else, I don't think we should start here. (Whether we should make everything translatable is an interesting question).

Also, in python3 it should be fine using the degree symbol in a string.

`rotate_0 = QAction(translate("0°"), rotate_angles_group)`

Reply:
Fair enough, I used it as that's how its structured in the ViewBox module so kept it consistent with that. If we start getting a non-english user base then that would be a good question to ask!